redis-utils-lib/src/cmr/redis_utils/embedded_redis_server.clj

```diff
@@ -0,0 +1,57 @@
+(ns cmr.redis-utils.embedded-redis-server
+  "Used to run an in memory redis server."
+  (:require
+   [cmr.common.lifecycle :as lifecycle]
+   [cmr.common.log :refer [debug info]]
+   [cmr.redis-utils.config :as redis-config])
+  (:import
+   (redis.embedded RedisExecProvider
+                   RedisServer)
+   (redis.embedded.util OS)))
+
+(def ^:private redis-server-executable-path
```

Comment:
Should this use `redis-install-path` from `project.clj` instead of assuming `redis` as the install directory?

Reply:
Ok so my clojure is a bit rusty but I believe that the src namespace's have no knowledge of the `project.clj` defined variables.